### PR TITLE
Add support to Intel Aero RTF

### PR DESCRIPTION
**Pull request by lucasdemarchi**
This changes the board name from "asc" that was used for the bring up to aerofc and finishes the bring up of Intel Aero FC. This flight controller will be available on the Intel Aero RTF.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Please check the output of the CI systems.

**Comment by lucasdemarchi** [comment]
```
/opt/gcc/bin/../lib/gcc/arm-none-eabi/4.9.3/../../../../arm-none-eabi/bin/ld: region `flash' overflowed by 2000 bytes
```

It's failing to build px4fmu-v2 for non obvious reasons since this PR shouldn't change the size for px4fmu-v2. I'll try to reproduce locally.

File: src/drivers/tap_esc/tap_esc.cpp [hunk 1]
```diff
@@ -262,6 +262,9 @@ TAP_ESC::init()
 		return ret;
 	}
 
+#ifdef CONFIG_ARCH_BOARD_AEROFC_V1
```
Comment (davids5) [inline src/drivers/tap_esc/tap_esc.cpp]: @lucasdemarchi 

We are moving away from having Board specific #if defs in the code. 

What is the reason for this?

We can discuss on skype if need be. skype:david_s5

Comment (lucasdemarchi) [inline src/drivers/tap_esc/tap_esc.cpp]: This needs to be revisited when final HW is out there. Right now it just returns 0 when trying to read back the values.

Comment (lucasdemarchi) [inline src/drivers/tap_esc/tap_esc.cpp]: I can add a comment explaining and leaving a TODO. It doesn't make much sense... but at least it allows to fly with our version of the hardware.

**Comment by bkueng** [comment]
@lucasdemarchi: these are the parameters we found after tuning the Aero RTF:
```
param set MC_ROLL_P 8.0000
param set MC_ROLLRATE_P 0.1300
param set MC_ROLLRATE_D 0.0012
param set MC_PITCH_P 8.0000
param set MC_PITCHRATE_P 0.1300
param set MC_PITCHRATE_D 0.0012
```

**Comment by lucasdemarchi** [comment]
@bkueng have you also tried with the tuning from  https://github.com/PX4/Firmware/pull/5893/commits/6d6f8a8a8cdd67470198057c6adeebc251e30676 ?

I'm not sure which one is better.  If yours is better, I can update that commit

**Comment by bkueng** [comment]
@lucasdemarchi Yes we used the ones in 6d6f8a8 as starting point and ended up changing to the ones I provided above.

**Comment by lucasdemarchi** [comment]
@bkueng updated the gains, thanks.

@LorenzMeier same error on other PRs, so I don't think the CI failures are related to this PR specifically.

**Comment by dagar** [comment]
@lucasdemarchi it's a flash overflow on px4fmu-v2 and a few trivial formatting fixes.

**Comment by lucasdemarchi** [comment]
@dagar have you seen what I said in https://github.com/PX4/Firmware/pull/5893#issuecomment-262200143 ?
I fixed the style errors, but I can't see where the flash overflow is coming from since I'm not touching anything for it except ~3 lines in mavlink.... and it's also failing for other PR.

**Comment by dagar** [comment]
It's so close to the limit the init script additions probably pushed it over. Should be lots of room after @julianoes merges https://github.com/PX4/Firmware/pull/5899

**Comment by LorenzMeier** [comment]
Will now pass once rebased.

**Comment by LorenzMeier** [comment]
Continued here: https://github.com/PX4/Firmware/pull/5916

**Pull request closed without merging**